src/main/java/org/systemexception/lunarlander/main/TestJbox.java

```diff
@@ -21,62 +29,149 @@
 public class TestJbox {
 
 	private static final String WINDOW_TITLE = "Physics in 2D!";
-	private static final int[] WINDOW_DIMENSIONS = {640, 480};
-
-	private static final World world = new World(new Vec2(0, -9.8f));
-	private static final Set<Body> bodies = new HashSet<Body>();
-
-	private static void render() {
-		glClear(GL_COLOR_BUFFER_BIT);
-		for (Body body : bodies) {
-			if (body.getType() == BodyType.DYNAMIC) {
-				glPushMatrix();
-				Vec2 bodyPosition = body.getPosition().mul(30);
-				glTranslatef(bodyPosition.x, bodyPosition.y, 0);
-				glRotated(Math.toDegrees(body.getAngle()), 0, 0, 1);
-				glRectf(-0.75f * 30, -0.75f * 30, 0.75f * 30, 0.75f * 30);
-				glPopMatrix();
-			}
+	private static final int[] WINDOW_DIMENSIONS = {800, 600};
+
+	private final String wall = "WALL", boxBody = "BOX", boxHeadBody = "BOXHEAD";
+	private final static double TWO_PI = 2 * Math.PI;
+
+	private final World world = new World(new Vec2(0, 9.8f));
+	private final HashMap<String, Body> bodies = new HashMap<>();
+
+	private TrueTypeFont font;
+
+	public static void main(String[] args) {
+
+		TestJbox testJbox = new TestJbox();
+		testJbox.start();
+	}
+
+	public void start() {
+		initGL();
+		initFonts();
+		setUpObjects();
+
+		while (!Display.isCloseRequested()) {
+			GL11.glClear(GL11.GL_COLOR_BUFFER_BIT);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'GL11.glClear' due to existing import 'org.lwjgl.opengl.GL11'](https://www.codacy.com/app/leo_4/lunar-lander/file/1690538463/issues/source?bid=3040650&fileBranchId=3043029#l54)